src/material/sort/sort-header.ts

```diff
@@ -127,7 +127,16 @@ export class MatSortHeader extends _MatSortHeaderBase
   @Input('mat-sort-header') id: string;
 
   /** Sets the position of the arrow that displays when sorted. */
-  @Input() arrowPosition: 'before' | 'after' = 'after';
+  @Input()
+  get arrowPosition(): 'before' | 'after' { return this._arrowPosition; }
```

Comment:
It might be a good idea to declare a type for `'before' | 'after'` so we don't have to repeat it.

Reply:
Done.